scenario/LiquidationBotScenario.ts

```diff
@@ -137,13 +292,382 @@ for (let i = 0; i < MAX_ASSETS; i++) {
         poolFee: daiPool.poolFee
       });
 
+      // confirm that Albert position has been abosrbed
       expect(await comet.isLiquidatable(albert.address)).to.be.false;
       expect(await comet.collateralBalanceOf(albert.address, collateralAssetAddress)).to.eq(0);
 
-      // XXX confirm that protocol reserves have increased
-
```

Comment:
We should also confirm that not all the collateral has been purchased.

Reply:
added a check that there are at least 10 units of the collateral asset after the partial liquidation